docs/design/input.md

```diff
@@ -7,29 +7,32 @@ Github audit plugin will be using config.yaml file for taking input from user
 ## provides logging level <OPTIONAL> , Default: info
 loglevel: debug
 ## folder path to log file <OPTIONAL> , Default: same as git-audit binary location 
-logpath: ./test/ 
+logpath: ./test.log 
 auditJobs:
 ## audit job name <REQUIRED>
 - name: auditjob1
-## polling interval to fetch data to be defined in cron job format  <REQUIRED> , Default: 5 * * * *   
-  polling_interval: 5 * * * *   
+## polling interval to fetch data to be defined  <REQUIRED> , Default: 5s, 10m etc  
+  polling_interval: 5m 
 ## metadata if any required like tags etc
   metadata:
   tags:
     tag1: tag1value
   ## git saas provider like github,bitbucket etc <REQUIRED>
-  repo_type: github
+  repo_host: github
   ## git repository name  <REQUIRED>
-  repo_name: testRepo   
+  repo_name: testRepo
+  ## git repository owner  <REQUIRED>
+  repo_owner: testOwner   
   repo_config:
   ## absolute url of repository <REQUIRED>
-    repo_url: https://www.github.com/test/testRepo   
-    ## credentials to access repository data <REQUIRED>
+    repo_url: https://www.github.com/test/testRepo
+    ## private or public repository <REQUIRED>
+    repo_type: public      
+    ## credentials to access repository data <REQUIRED for private repo>
     credentials:  
-      ## either email or username is required    <REQUIRED>
+      ## username is required    <REQUIRED for private repo>
       username: testRepo  
-      email: restRepo@test.com
-      ## API token in base64 encode format. <REQUIRED> , cannot be empty
+      ## API token in base64 encode format. <REQUIRED for private repo> , cannot be empty
```

Comment:
Token can be used for public repo as well, without token github applies API rate limit

Reply:
should i add it mandatory for public also